epoxy-adapter/src/main/java/com/airbnb/epoxy/ChangeAnimationDelegator.java

```diff
@@ -153,16 +203,58 @@ public boolean isRunning() {
     return changesRunning || super.isRunning();
   }
 
+  @Override
   public boolean canReuseUpdatedViewHolder(@NonNull ViewHolder holder,
       @NonNull List<Object> payloads) {
     // pass empty list to super because it contains only the previous model and DefaultItemAnimator
     // would decide not to cross fade
     return (holder instanceof EpoxyViewHolder && !payloads.isEmpty())
-        || super.canReuseUpdatedViewHolder(holder, Collections.emptyList());
+        || super.canReuseUpdatedViewHolder(holder,
+        Collections.emptyList()); //TODO: only remove our payloads
```

Comment:
Based on the `could not get previousModel from change payloads` error below it seems like you want to only support EpoxyController